lte/gateway/c/session_manager/sessiond_main.cpp

```diff
@@ -106,6 +108,12 @@ static magma::mconfig::SessionD load_mconfig() {
   return mconfig;
 }
 
+static magma::mconfig::SharedMconfig load_shared_mconfig() {
```

Comment:
Should we move this function into the mconfig library so that it can be shared with other services?

Reply:
Yes. I would refactor this within the follow up PR handling MME. I realized that MME is pure C code, which does not allow for some C++ features used here (e.g. objects). So a change to generalize will anyway be necessary for the MME effort.